customersupport/controllers/ordersearch.py

```diff
@@ -1,26 +1,70 @@
 from flask import jsonify
+from flask import request
 
 from customersupport import app
 from customersupport.wrappers import sales
 
+def get_param_from_request_if_not_empty(param_name):
+    """Get a query parameter from the request, or None if the parameter is empty or missing."""
+    value = request.args.get(param_name)
+    if value is not None and value != "":
+        return value
+    else:
+        return None
 
-@app.route('/sales/ordersearch/real')
-def search_order(mock=False):
-    orders = sales.get_orders(
-        customer_id=1,
-        incl_items=True,
-        mock=mock
-    )
+# select which order search to use
+# if order id exists use order search
+# otherwise order info
+@app.route('/api/order/search')
+def select_search():
```

Comment:
Not a blocker, but the docstrings should be inside the function in `"""` blocks to match [Python's docstring format](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).